index.Rmd

```diff
@@ -137,15 +137,15 @@ Now I can add all bands together to have a metadata containing all informations
 
 For Earth data, I decided to add only band2, band3 and band7 together because those wavelength are able to reflect the distribution of water on the sites. Also I just used those bands because the code does not work when I add the remaining band to the current I already stack together.
 
-However, only the Martian data is important here because we are working on Mars. So  for Martian data, we have the same size so we will just add them all together.But I add only band4, band5, band6, band8 and band9 together because those wavelength are able to reflect the distribution of water on the sites. Also I just used those bands because the code does not work when I add the remaining band to the current I already stack together.
+However, only the Martian data is important here because we are working on Mars. So, for Martian data, we have the same size so we will just add them all together. But I add only band4, band5, band6, band8 and band9 together because those wavelength are able to reflect the distribution of water on the sites. Also I just used those bands because the code does not work when I add the remaining band to the current I already stack together.
 ```{r}
 image1 <- stack(Band2, Band3, Band7) #Here is how it works for Landsat data on Earth
-#image1 <- stack(band4, band5, band6)
+#image1 <- stack(band4, band5, band6) #Clear specification of what this line is for and why it was commented out
```

Comment:
You nicely specified in the lines above and below what they were for, but if you used a comment to specify whey you commented out line 143, that would be helpful.